Sort events in a more stable manner

bryanl opened the pull request:
#404

wwitzel3 commented on build.go:
```diff
@@ -135,6 +137,10 @@ func build() {
 }
 
+func runDev() {
+	runCmd("build/octant", nil)
```
Can you make this check for the existence of `build/octant` and run `build` if it isn't find?

bryanl replied:
I updated the code and split the commits. Now `newCmd` will check for the existence of the command first.

wwitzel3 reviewed (approved):
This works nicely for me, I made a suggestion, but not a blocker

bryanl committed:
Sort events in a more stable manner

Signed-off-by: bryanl <bryanliles@gmail.com>

bryanl committed:
Add command to run octant from the build directory

* Updates newCmd to exit if command is not found
* Adds a run-dev command

Since build supports multiple commands, you can `go run build.go build run-dev`

Signed-off-by: bryanl <bryanliles@gmail.com>
